tests_v2/test_browser_api_comprehensive.py

```
@@ -139,23 +139,65 @@ def _build_pages() -> tuple[str, str]:
+            path = self.path.split("?", 1)[0]
+            body = encoded_pages.get(path)
+            if body is None:
+                self.send_error(404)
+                return
+            self.send_response(200)
+            self.send_header("Content-Type", "text/html; charset=utf-8")
+            self.send_header("Content-Length", str(len(body)))
+            self.end_headers()
+            self.wfile.write(body)
+
+        def log_message(self, _format: str, *_args: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*_args`

(ANN401)